### ISSUE TITLE
Running eralchemy gives error

### ISSUE THREAD

**Issue by fortm** [Alexis-benoist/eralchemy#28]
Running eralchemy gives following error on Win 10 

Please install application. using "pip install application.".

I have already installed these ->
1. Install python
2. Visual C++ for python -> https://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi
3. graphviz -> http://www.graphviz.org/Download_windows.php
4. pygraphviz -> pip install --global-option=build_ext --global-option="-IC:\Program Files (x86)\Graphviz2.38\include" --global-option="-LC:\Program Files (x86)\Graphviz2.38\lib\release\lib" pygraphviz
5. eralchemy -> pip install eralchemy 
6. Install instant client for oracle -> http://www.oracle.com/technetwork/topics/winx64soft-089540.html
7. pip install cx_Oracle-5.3+oci12c-cp27-cp27m-win32.whl after downloading from http://www.lfd.uci.edu/~gohlke/pythonlibs/#cx_oracle

**Comment by Alexis-benoist** [comment]
I'm using a OSX and I don't know.
If someone knows :)

**Comment by klondikemarlen** [comment]
I had a similar error that was "pip install directory". I fixed it by reinstalling the driver for my database. In my case that was `pip install --upgrade --force-reinstall mysql-python`. Only, of course, after reinstalling every other module and application that might possibly have been related :P

NOTE: I did eventually give up on making ERAlchemy work on native Windows 10 ... and ended up installing and using it through Bash on Ubuntu on Windows 10. I posted a guide https://github.com/Alexis-benoist/eralchemy/issues/47. It is actually much faster and easier to use Windows that way too.
